src/main/java/newbank/server/commands/Command.java

```diff
@@ -6,6 +6,13 @@
 /** Abstract representation of a command. */
 public abstract class Command {
   public abstract String execute();
+  
+  /**
+   * @return the command syntax
+   */
+  protected String getSyntax() {
```

Comment:
Shouldn't this be an abstract method, to avoid the need for this redundant implementation?

Reply:
That's the intent. However this needs a separate item.

Reply:
A separate item?